util/topgen/lib.py

```diff
@@ -495,3 +495,15 @@ def make_bit_concatenation(sig_name: str,
         acc += [',', item_indent, item]
     acc += ['\n', ' ' * end_indent, '}']
     return ''.join(acc)
+
+
+def param_value (ip_param, module_params_decl):
```

Comment:
Nit: PEP8 style (which we claim to follow) has no space before the opening paren

Reply:
thanks! i should have noticed the lint failure.